include/nuttx/video/rgbcolors.h

```diff
@@ -33,10 +33,9 @@
 
 /* Color Creation and Conversion Macros *************************************/
 
-/* This macro creates RGB24 from 8:8:8 RGB */
-
```

Comment:
why change

Reply:
It doesn't need to be there - I forgot to delete it after I'd removed the dependency on rgbcolors.

Reply:
Comment here was incorrect - I didn't need to change rgbcolors.h in the end, but didn't back out all changes including a style change (now, I understand, that shouldn't be done in this submission) and deleting a line I shouldn't have.

Reverted/fixed.